### ISSUE TITLE
When label position is set to hidden, the label element is not rendered

### ISSUE THREAD

**Issue by chrismlusk** [verbb/formie#1638]
### Describe the bug

If I set the **Label Position** option to hidden, then the `<label>` element is missing from the DOM. I first noticed this in a Group field, but I can reproduce with other field types, e.g., Email, Single-Line Text.

I do have a project-specific form template, but it does not use custom templates (i.e., the "Use Custom Template" option is not enabled). But, I also tested by updating to "Default Formie Template" under Appearance, and the issue remained.

### Steps to reproduce

Update a field's appearance to set Label Position to Hidden:

<img width="528" alt="settings" src="https://github.com/verbb/formie/assets/5922913/0b18d877-8b61-4ca4-864f-6e6ad8968a88">

Inspect the rendered markup and notice the label is no longer included in the field container.

```html
<div class="fui-row fui-page-row">
  <div class="fui-field fui-type-email-address fui-label-hidden fui-subfield-label-hidden fui-instructions-above-input fui-field-required fui-page-field" data-field-handle="email" data-field-type="email-address">
    <div class="fui-field-container">
      <!-- Note: Label is no longer rendered here... -->
      <div class="fui-input-container">
          <input type="email" id="fui-demo-owaeak-fields-email" class="fui-input" name="fields[email]" autocomplete="email" required data-fui-id="demo-email">
      </div>
    </div>
  </div>
</div>
```

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.5.11

### Plugin version

2.0.42

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.42"`.

**Comment by engram-design** [comment]
Fixed in [2.0.43](https://github.com/verbb/formie/releases/tag/2.0.43)
